packages/core/src/Point.ts

```diff
@@ -19,7 +19,7 @@ export class Point {
   private tags: {[key: string]: string} = {}
   /** escaped field values */
   public fields: {[key: string]: string} = {}
-  private time: string | number | Date | undefined
+  private time: string | number | Date | unknown | undefined
```

Comment:
Same as above comment, this reduces to just `unknown`, which is probably not what you want.

Reply:
This redundancy is intentional, I'll keep it.